app/Models/BookRepository.php

```
@@ -66,6 +68,12 @@ public function getById(int $id): ?array
             $sql .= ", NULL AS sottogenere_nome";
         }
 
+        if ($hasSeriesHierarchy) {
+            $sql .= ", c.gruppo_serie, c.ciclo AS ciclo_serie, c.ordine_ciclo, c.tipo AS tipo_collana, cp.nome AS serie_padre";
+        } else {
+            $sql .= ", NULL AS gruppo_serie, NULL AS ciclo_serie, NULL AS ordine_ciclo, NULL AS tipo_collana, NULL AS serie_padre";
+        }
```

PHPMD (2.15.0)

[error] The method getById uses an else expression. Else clauses are basically not necessary and you can simplify the code by not using them. (undefined)

(ElseExpression)